### DISCUSSION TITLE
Tailwind CSS not recognizing $mantine-breakpoint-md variable in CSS modules

### DISCUSSION THREAD

**Discussion by fecoderchinh** [mantinedev/mantine#5256] [Q&A]
### Features of the template, describe how it is different from other templates

Hello,
I tried to use tailwindcss within css modules.
Here is my code
```
// Welcome.module.css
.title {
  color: light-dark(var(--mantine-color-black), var(--mantine-color-white));
  font-size: rem(100px);
  font-weight: 900;
  letter-spacing: rem(-2px);

  @media (max-width: $mantine-breakpoint-md) {
    font-size: rem(50px);
  }

@apply flex;
}
```
At the present, everything work fine but in VSCode, i got a message `term expectedtailwindcss(css-termexpected)` as tailwindcss does not know `$` prefix `$mantine-breakpoint-md`

I don't know if this will cause any problem in the future so i create this topic.

Does anyone know how to fix?

**Comment by idindrakusuma** [comment]
Try this one @fecoderchinh 
https://mantine.dev/getting-started/#set-up-vs-code

**Reply by fecoderchinh** [reply]
Hello @idindrakusuma , 
Sorry for late reply, i have tried your suggestion but it does not work,
![image](https://github.com/mantinedev/mantine/assets/13318452/43ba0c4c-aa58-4332-b1fe-fbb137445de6)
here is my `.vscode/settings.json`

```
"emmet.includeLanguages": {
    "postcss": "css"
},
"cssVariables.lookupFiles": [
    "**/*.css",
    "**/*.scss",
    "**/*.sass",
    "**/*.less",
    "node_modules/@mantine/core/esm/index.css"
],
```

**Reply by Jdruwe** [reply]
Did you also install [PostCSS Intellisense and Highlighting](https://marketplace.visualstudio.com/items?itemName=vunguyentuan.vscode-postcss)?
